research/evm_explorer_comparison.md

```diff
@@ -0,0 +1,152 @@
+	- Etherscan's well-structured documentation makes it easy for developers to dive in and leverage blockchain data for their projects.
+
+4. Analytics and Verification Services:
+	- It allows users to verify smart contracts and token information.
+	- Users can access various analytics and charts related to network performance and usage.
+
+**Pros**
+- **Reputation and Trust:**
+
+	Etherscan is considered by many Ethereum explorers to be the industry standard. Because of its extended stay in the Ethereum ecosystem, it has established a solid reputation for accuracy and dependability. Because of the community's confidence, Etherscan is the go-to option for developers looking for a trustworthy source of blockchain data.
+- **Detailed Information:**
+
+	When it comes to providing detailed and accurate blockchain data, Etherscan shines. It provides comprehensive details about tokens, blocks, smart contracts, and transactions. Developers get access to comprehensive data points, such as transaction statuses, timestamps, and gas fees. For the purpose of troubleshooting and assessing blockchain activity, this level of information is essential when building a new type of scaling solution.
+
+- **Developer Support:**
+
+	Etherscan offers a robust suite of [APIs](https://sebs.github.io/etherscan-api/) and [tools](https://etherscan.io/directory/Others/Tools) tailored for developers. The APIs provide access to a wide range of data, from basic transaction details to advanced analytics. Developers can integrate Etherscan's services into their applications to enhance functionality. Additionally, the Etherscan team offers thorough documentation and responsive support, ensuring that developers can effectively utilize these resources.
+
+- **User Experience:**
+	Etherscan's user interface is designed with both novice and experienced users in mind. The interface is intuitive, well-organized, and easy to navigate, allowing users to quickly find the information they need. Features like search functionalities, customizable views, and clear data presentation contribute to a seamless user experience, which is particularly beneficial when dealing with complex blockchain data.
+
+- **Additional Services:**
+	Beyond basic blockchain exploration, Etherscan offers a suite of additional features that enhance its utility. Such features are: [Dex tracker](https://etherscan.io/dex), [Gas tracker](https://etherscan.io/gastracker), [Token tracker](https://etherscan.io/tokens), [Smart contract verification](https://etherscan.io/verifyContract), [Developer tools](https://etherscan.io/directory/Others/Tools).
+
+**Cons**
+- **Proprietary:**
+
+	Etherscan is not open-source, hence its code is not publicly available for anyone to inspect or modify. For developers, this lack of transparency can be a drawback because it limits the ability to customize the explorer to suit specific needs or integrate it more deeply with their own projects.
+
+- **Ethereum-Centric:**
```

Comment:
I don't think this is a disadvantage. AFAIK Blockscout is also Ethereum-Centric only. I suggest removing this point.